modules/heat_conduction/src/bcs/GapHeatTransfer.C

```diff
@@ -194,8 +190,9 @@ GapHeatTransfer::computeQpJacobian()
   if (!_has_info)
     return 0.0;
 
-  return _test[_i][_qp] * ((_u[_qp] - _gap_temp) * _edge_multiplier * _gap_conductance_dT[_qp] +
-                           _edge_multiplier * _gap_conductance[_qp]) *
+  return _test[_i][_qp] *
+         ((_u[_qp] - _gap_temp) * _edge_multiplier * _gap_conductance_dT[_qp] +
+          _edge_multiplier * _gap_conductance[_qp]) *
```

Comment:
While you're in here...shouldn't _edge_multiplier be multiplied only once here? Residual is
```
(_u[_qp] - _gap_temp) * _edge_multiplier * _gap_conductance[_qp]
```

Reply:
wait scratch that...I didn't see the +